frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -295,6 +304,11 @@ class Receive extends Component<Props, State> {
                             <div className="m-bottom-half">
                                 <CopyableInput value={address} />
                             </div>
+                            { receiveAddresses.length > 1 && (
+                                <p className={style.changeType} onClick={this.toggleAddressType}>
+                                    {t(`receive.addressType.${this.state.addressType}`)}
+                                </p>
+                            )}
```

Comment:
It would be great to have the addressType toggle inside the verify dialog, but at that point the BB02 is already in address confirmation mode (obviously).
@benma is there any way we can toggle the addresstype at this point?…
if not it is not a big deal, then I move the toggle into the receive view, before the dialog.

Reply:
Currently it is not possible to automatically change the screen on the bitbox02 once it is showing an address. The user has to interact with the device.

This is gonna be fixed at some point, but we're not there yet.